src/App.jsx

```diff
@@ -1,13 +1,214 @@
-
 import './App.css'
+import {useState} from "react";
+import FruitCounter from "./components/FruitCounter.jsx";
+import FormInputField from "./components/FormInputField.jsx";
 
 function App() {
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    const [bananas, setBananas] = useState(0);
+    const [strawberries, setStrawberries] = useState(0);
+    const [apples, setApples] = useState(0);
+    const [kiwis, setKiwis] = useState(0);
+
+    function resetAllFruits() {
+        setStrawberries(0);
+        setBananas(0);
+        setApples(0);
+        setKiwis(0);
+    }
+
+    const [firstname, setFirstname] = useState("");
+    const [lastname, setLastname] = useState("");
+    const [age, setAge] = useState(0);
+    const [postalcode, setPostalcode] = useState("");
+    const [frequency, toggleFrequency] = useState("Wekelijks");
+    const [timeslot, toggleTimeslot] = useState("Overdag");
+    const [remarks, setRemarks] = useState("");
+    const [agreeTerms, toggleAgreeTerms] = useState(false);
+
+    function submitForm(e) {
+        e.preventDefault();
+        console.log(`Details bestelling van ${firstname} ${lastname}`);
+        console.log(`Bestelde producten: Bananen (${bananas}X), aardbeien (${strawberries}X), appels (${apples}X) & kiwi's (${kiwis}X)`);
+        console.log(`
+        Voornaam: ${firstname},
+        Achternaam: ${lastname},
+        Leeftijd: ${age},
+        Postcode: ${postalcode},
+        Bezorg frequentie: ${frequency},
+        Bezorg tijdslot: ${timeslot},
+        Opmerkingen: ${remarks},
+        Algemene voorwaarden: ${agreeTerms}`
+        );
+    }
+
+    return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+
+            <section className="productOverview">
+
+                <h2>Onze producten</h2>
+                <article className="productCard">
+                    <h3>🍌 Bananen</h3>
```

Comment:
Dit zie ik per fruitsoort herhaald worden. Dit had je dus óók op kunnen nemen in het FruitCounter component om code dubbeling verder te reduceren